artifactory/commands/transferfiles/state/timeestimation.go

```diff
@@ -0,0 +1,228 @@
+package state
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles/api"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const (
+	secondsInMinute            = 60
+	secondsInHour              = 60 * secondsInMinute
+	secondsInDay               = 24 * secondsInHour
+	milliSecsInSecond          = 1000
+	bytesInMB                  = 1024 * 1024
+	bytesPerMilliSecToMBPerSec = float64(milliSecsInSecond) / float64(bytesInMB)
+	// Precalculated average index time per build info, in seconds.
```

Comment:
I think we should elaborate more here, explaining about how this value is used, allowing the random reader to get better understanding around the goal.